src/commonMain/kotlin/com/github/michaelbull/result/Binding.kt

```diff
@@ -38,11 +38,7 @@ inline fun <V, E> binding(crossinline block: ResultBinding<E>.() -> V): Result<V
     }
 }
 
-internal object BindException : Exception() {
-    override fun fillInStackTrace(): Throwable {
-        return this
-    }
-}
```

Comment:
Given this change, surely we need to change this to an `expect` and `actual` contract, i.e. `expect class BindException` to exist and on the JVM one explicitly override the `fillInStackTrace()` in an `actual class BindException`? Otherwise we lose the behaviour of stacktrace avoidance

Reply:
ah good thinking!

Reply:
done